sites/kit.svelte.dev/src/routes/faq/index.svelte

```diff
@@ -0,0 +1,137 @@
+<script context="module">
+	export async function load({ fetch }) {
+		const faqs = await fetch('faq.json').then(r => r.json());
+		return { props: { faqs } };
+	}
+</script>
+
+<script>
+	export let faqs;
+</script>
+
+<svelte:head>
+	<title>FAQ • SvelteKit</title>
+	<meta name="twitter:title" content="SvelteKit FAQ" />
+	<meta name="twitter:description" content="The fastest way to build Svelte apps" />
+	<meta name="Description" content="The fastest way to build Svelte apps" />
+</svelte:head>
+
+<div class="faqs stretch">
+	<h1>Frequently Asked Questions</h1>
+	{#each faqs as faq}
+		<article class="faq">
+			<h2>
+				<a id={faq.slug} class="offset-anchor" />
+				<a class="anchor" sapper:prefetch href="faq#{faq.slug}" title={faq.title}>&nbsp;</a>
+				{faq.metadata.question}
+			</h2>
+			{@html faq.html}
+		</article>
+	{/each}
+	<p>
+		See also the <a href="https://svelte.dev/faq" rel="external">Svelte FAQ</a> for questions relating
+		to Svelte directly.
+	</p>
+</div>
+" /> -->
```

Comment:
this looks like an extra line that should be deleted? I don't see the start that would match up to it and it doesn't appear in the Sapper version: https://github.com/sveltejs/sapper/blob/master/site/src/routes/faq/index.svelte

Reply:
Not sure why that was added, some auto-correct via Prettier I missed, sorry!